platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/ConsensusImpl.java

```diff
@@ -664,6 +679,27 @@ private List<EventImpl> getStronglySeenInPreviousRound(final EventImpl event) {
                 getMinRoundGeneration(),
                 Math.max(previousRoundNonExpired, decidedRoundNumber - config.roundsExpired() + 1));
 
+        // Check for no judges or super majority conditions.
+        final long judgeWeights = judges.stream()
+                .mapToLong(event -> getWeight(event.getCreatorId()))
+                .sum();
+        consensusMetrics.judgeWeights(judgeWeights);
+        if (judges.isEmpty()) {
+            markerFileWriter.writeMarkerFile(NO_JUDGES_MARKER_FILE);
```

Comment:
We *always* expect to have judges, right? If so, maybe this should log an error too

Reply:
added rate limited logger for an error here.